android/demo/src/androidTest/java/com/intuit/player/android/reference/demo/test/ApplitoolsMockTest.kt

```diff
@@ -1,64 +0,0 @@
-package com.intuit.player.android.reference.demo.test
-
-import android.content.Intent
-import android.net.Uri
-import androidx.test.core.app.ApplicationProvider
-import androidx.test.espresso.matcher.ViewMatchers.withId
-import androidx.test.ext.junit.rules.activityScenarioRule
-import androidx.test.filters.LargeTest
-import com.applitools.eyes.android.espresso.fluent.Target.region
-import com.intuit.player.android.reference.demo.R
-import com.intuit.player.android.reference.demo.model.AssetMock
-import com.intuit.player.android.reference.demo.test.base.ApplitoolsTest
-import com.intuit.player.android.reference.demo.ui.main.MainActivity
-import com.intuit.player.android.reference.demo.ui.main.MainViewModel
-import com.intuit.player.jvm.utils.mocks.ClassLoaderMock
-import com.intuit.player.jvm.utils.mocks.Mock
-import com.intuit.player.jvm.utils.mocks.getFlow
-import org.junit.Rule
-import org.junit.Test
-import org.junit.runner.RunWith
-import org.junit.runners.Parameterized
-import java.net.URLEncoder
-
-@LargeTest
-@RunWith(Parameterized::class)
-class ApplitoolsMockTest(val mock: Mock<*>?) : ApplitoolsTest() {
```

Comment:
we talked about removing this a while ago since this only took a screenshot for each mock as it renders and does nothing else, most of those screenshots are duplicates, + increasing the CI time